ee/codegen/src/project.ts

```diff
@@ -175,7 +178,17 @@ ${errors.slice(0, 3).map((err) => {
 
   public async generateCode(): Promise<void> {
     const assets = await this.generateAssets().catch((error) => {
-      this.workflowContext.addError(error);
+      if (error instanceof BaseCodegenError) {
+        this.workflowContext.addError(error);
+      } else {
+        console.error("Unexpected error generating assets", error);
```

Comment:
I looked into creating a test for this, but I couldn't figure out how to repro based on the sentry error. So am hoping to get more info from this console.error